src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -218,8 +217,7 @@ gl_sarray activity_classifier::predict(gl_sframe data,
       gl_sframe({{"temp", duplicated_preds_per_window}}).stack("temp", "preds");
 
   gl_sarray result = preds_per_row["preds"];
-
-  if (output_type == "class") {
+    if (output_type == "class") {
```

Comment:
This statement should also remain at its previous indent level, the same as the prior statement